### PR TITLE
Update paper

### PR DESCRIPTION
**Pull request by johandahlberg**
As suggested in the JOSS review (https://github.com/openjournals/joss-reviews/issues/556), I'm adding a reference to basecallQC into the paper.

I'm also bumping the version to 1.3.0.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ contains a feature of setting thresholds for different QC criteria based on inst
 type and evaluates these criteria automatically, which to the best of our knowledge, is not supported
 by existing software.
 
+Another software library which focuses on working with Illumina data from a sequencing run perspective
+is basecallQC [@basecallQC]. This is a library for the R [@R] programming language, which allows the user
+to perform utility actions related to a sequencing run, as well as create summary reports.
+It does not, however, provide built in functionality to access quality control thresholds, nor does it
```
Comment (matrulda) [inline paper/paper.md]: Suggestion: "It does not, however, provide built in functionality to assess whether quality control thresholds are fulfilled or not, nor does it [...]"

**Review by matrulda — COMMENTED** [review]
Nice! I have suggested a change in the text. Take a look.

**Comment by johandahlberg** [comment]
Great suggestion. I've fixed it now.

**Review by matrulda — APPROVED** [review]
Let me know if you want it merged @johandahlberg

**Comment by johandahlberg** [comment]
You can go ahead and merge. 👍

**Pull request merged**